exporter/collector/metrics_test.go

```diff
@@ -495,7 +631,7 @@ func TestExponentialHistogramPointToTimeSeries(t *testing.T) {
 	// Add a second point with no value
 	hist.DataPoints().AppendEmpty().SetFlags(pmetric.MetricDataPointFlags(pmetric.MetricDataPointFlagNoRecordedValue))
 
-	tsl := mapper.exponentialHistogramToTimeSeries(mr, labels{}, metric, hist, point)
+	tsl := mapper.metricToTimeSeries(mr, labels{}, metric)
```

Comment:
Note to reviewers: this test was broken, in that it wasn't checking the second point with the "NoRecordedValue" flag at all.  This change fixes it.